tailwind.config.js

```diff
@@ -1,15 +1,20 @@
 module.exports = {
   purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
-  darkMode: false, 
+  darkMode: false,
   theme: {
+    // compiled the css with the below primary and secondary colors and linked with _app.tsx.
+    colors: {
+      primary: '#C54582',
+      secondary: '#F4DCF4',
+    },
     extend: {
-      backgroundImage: theme => ({
+      backgroundImage: (theme) => ({
```

Comment:
can we format all files in the project as a commit so that we don't see formatting differences again for any other PR. This change is ok for now